doc/_shared/options/from-repo.rst

```diff
@@ -0,0 +1,3 @@
+``--from-repo=REPO_ID,...``
+    | The following items can be selected only from the specified repositories. All enabled repositories will still be used to satisfy dependencies.
```

Comment:
This is the same  text as in the --help output. Wouldn't be better to use different, probably more detailed, explanation what the option does? It would be also good to explain what happens if --from-repo names a repository which is not enabled.

E.g. "Packages or their provides directly named on the command line to operate on will only be looked up from these specified repositories. Enabling and disabling repositories still applies. However, dependencies of those packages will be resolved from any enabled repository."